src/count_code_lines.py

```diff
@@ -0,0 +1,60 @@
+"""Program to count the number of lines added and deleted by an indvividual."""
+
+# from git import Repo
+from pydriller import RepositoryMining
+
+# from pydriller.domain.commit import ModificationType
+
+
+def print_in_table(dictionary):
+    """Create the table."""
+    # print headings
+    print("Username", "\t +", "\t -", "\t Total")
+    # prints hashmap content
+    for key in dictionary:
+        print(
+            key,
+            "\t",
+            dictionary[key][0],
+            "\t",
+            dictionary[key][1],
+            "\t",
+            dictionary[key][2],
+        )
```

Comment:
This code works fine here. But maybe try to look into some libraries like `python-tabulate` or `prettytable` to formalize the way displaying tables

Reply:
Thanks for the suggestion, we will look further into that and try to use these helpful packages.